Add new logo

aaaberg opened the pull request:
### Description
- Adds the new logo suggested in #1. 
- Removes the old @1x and @2x logos from the repo.

Closes #1

aaaberg committed:
Delete extension.png

Removes the old logo

aaaberg committed:
Delete extension@2x.png

aaaberg committed:
add new extension.png file that contains new logo

aaaberg committed:
Merge pull request #1 from aaaberg/remove-old-logos

Delete old logos and add new logo

aaaberg committed:
Add files via upload

aaaberg committed:
Delete extension.png

aaaberg committed:
Upload proper extension.png

chrisdl commented:
@aaaberg https://docs.nova.app/extensions/images/#recommended-icon-sizes It looks like Nova recommends having the `@2x` and the 1x. Or are you proposing we do just the one image at 64x64 and then allow for scaling for it to go down to 1x at 32x32?

aaaberg committed:
Delete extension.png

aaaberg committed:
Add newly sized logo files to the repo

After discussion with @chrisdl we came to the conclusion to try 64 and 128 (1x, 2x) for sizes to see if it looks good. Balancing clarity and how much someone needs to download.

aaaberg commented:
@chrisdl, added the new logo sizes. Please have a look. :)

aaaberg commented:
@chrisdl - what happened to merging this one?
